code/game/machinery/doors/airlock.dm

```diff
@@ -1109,13 +1109,16 @@ About the new airlock wires panel:
 	if(istype(I, /obj/item/twohanded/fireaxe)) //let's make this more specific //FUCK YOU
 		var/obj/item/twohanded/fireaxe/F = I
 		if(F.wielded)
-			spawn(0)
-				if(density)
-					open(1)
+			if(density)
+				var/time_to_open = 75
```

Comment:
This could be moved to the `do_after()` proc, since it's not being used elsewhere.

Reply:
done!